snaplets/postgresql-simple/devel.cfg

```diff
@@ -0,0 +1,17 @@
+host = "localhost"
+port = 5432
+user = "eibarra"
+pass = "eibarra"
```

Comment:
:scream: 
This should be in an env variable... I think it's ok for now, I'll add a card in trello so you can read about configurable values and how to encrypt sensitive information... but you don't have to implement it here